Added vial keymap for coseyfannitutti/romeo

enemysfriend22 opened the pull request:
Added vial support for the Romeo40 from coseyfannitutti based on the via keymap. Includes all layout options. Tested only the split bars

xyzz commented:
There are unrelated changes under `keyboards/bioi/f60` and `keyboards/sneakbox/aliceclonergb/keymaps/vial/vial.json`

enemysfriend22 commented:
Sorry about that, new to all this. Will try to fix it in the morning.

enemysfriend22 committed:
Added vial keymap for romeo 40%

Added basic vial support for the Romeo40 from coseyfannitutti. Tested only the split bars

enemysfriend22 commented:
Hey, should be fixed now. Sorry for the inconvenience.
